test/spec-testing.js

```diff
@@ -0,0 +1,69 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/loopback-next-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+
+const Dredd = require('dredd');
+const expect = require('@loopback/testlab').expect;
+const HelloWorldApp = require('../lib/application').HelloWorldApp;
+const http = require('http');
+
+const port = 3030; // use this port for Dredd testing only
+const localhost = 'http://localhost:' + port;
+const dreddConfig = {
+  server: localhost + '/', // base path to the end points
+  options: {
+    level: 'verbose',
+    silent: true, // false for helpful debugging info
+    path: [localhost + '/openapi.json'], // to download apiSpec from the service
+  }
+};
+
+describe('Api Spec End Points', () => {
+  let server;
+  let error;
+  let ctFailures;
+  beforeEach(done => {
+    initEnvironment(dreddConfig, (dredd) => {
+      testApi(dredd, done);
+    });
+  });
+  afterEach(done => {
+    server.close();
+    done();
+  });
+
+  describe('Test end points', () => {
+    it('passes "req parameter and res result" match test', () => {
```

Comment:
So this makes the following test name: `Api Spec End Points Test end points passes "req parameter and res result" match test`. That does not make any sense to me.

I would like this file to have simpler structure:

```ts
describe('Application API', () => {
  before(initEnvironment);
  after(cleanEnvironment);

  it('implements the specification', done => {
    dredd.run(done);
  });
});
```